packages/admin/admin/src/rowActions/RowActionsItem.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+
+import { RowActionsIconItem, RowActionsIconItemComponentsProps, RowActionsIconItemProps } from "./RowActionsIconItem";
+import { RowActionsListItem, RowActionsListItemComponentsProps, RowActionsListItemProps } from "./RowActionsListItem";
+import { RowActionsMenuContext } from "./RowActionsMenu";
+
+export interface CommonRowActionItemProps {
+    icon?: React.ReactNode;
+    text?: React.ReactNode;
```

Comment:
I'd prefer the following API using `children`:

```tsx
<RowActionsItem startIcon={<Online />} {/* Could be endIcon as well */}>
  Additional Item 1
</RowActionsItem>
```

This would be similar to the MUI Button props, and would offer more flexibility. What do you think?

Reply:
Yes, `children` makes more sense here, I've changed that 👍🏼 

But I'd keep `icon` instead of changing it to `startIcon`.  
`endIcon` for example is only used at the end of the item, when rendered as a ListItem.  
`icon` on the other hand is either rendered at the beginning of the ListItem, or as the only icon, when rendered as an IconButton.